### PR TITLE
Deb package

### PR DESCRIPTION
**Pull request by apoorvam**
Added deb and rpm package release.
Used Dunner itself to make rpm release since travis does not support required version of rpmbuild.
Snap is released outside the docker since it is not compatible.

ToDo: 

* Add environment keys `BINTRAY_USER` and `BINTRAY_API_KEY`

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -44,14 +45,12 @@ nfpm:
   name_template: '{{ .ProjectName }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
   homepage:  https://github.com/leopardslab/Dunner
   description: A Docker based task runner tool
+  maintainer: Milindu Sanoj Kumarage <agentmilindu@gmail.com>
```
Comment (agentmilindu) [inline .goreleaser.yml]: Let's change this to LeaopardsLabs community

Comment (apoorvam) [reply .goreleaser.yml]: Do we have an email id for that? @agentmilindu

File: .snap_goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+project_name: dunner
+
+snapcraft:
+  name_template: '{{ .ProjectName }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+  summary: A Docker based task runner tool
+  description: |
+    Dunner is a task runner tool like Grunt but used Docker images like CircleCI do. | 
+    You can define tasks and steps of the tasks in your `.dunner.yaml` file and then run these steps with `Dunner do taskname`
+  grade: stable
+  confinement: classic
```
Comment (agentmilindu) [inline .snap_goreleaser.yml]: Shall we try again with `strict` confinement?

File: release/publish_deb_to_bintray.sh [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  rm -f "$(pwd)/*.deb"
+}
+
+cleanArtifacts
+downloadDebianArtifacts
+setVersion
+printMeta
+setUploadDirPath
+bintrayUpload
+snooze
```
[the remark this answers is not in the document]
Comment (apoorvam) [reply release/publish_deb_to_bintray.sh]: After publishing to bintray, it sometimes takes a second to reflect on site so we can mark them as default downloadable files. So snoozing for a few seconds.

File: .snap_goreleaser.yml (hunk 2 above)
Comment (apoorvam) [reply .snap_goreleaser.yml]: I updated with snap changes for strict. Please review now.

**Comment by apoorvam** [comment]
I have updated this PR as release with Goreleaser is working fine for rpm packages now. Since rpm is recommended to use in fedora or similar distros and deb on ubuntu/etc, its working as expected. 

We need to add BINTRAY keys as mentioned in travis env.

**Pull request merged**